app/views/admin/theft_alerts/edit.html.haml

```diff
@@ -6,7 +6,18 @@
     url: admin_theft_alert_path(@theft_alert),
     method: :patch,
     html: { class: "m-0" } do |f|
-    = hidden_field_tag :state_transition, params.fetch(:state_transition, "update_details")
+    .form-group
+      = f.label :status, class: "form-label"
+      = f.select :status, options_for_select(TheftAlert.statuses.keys), class: "form-control"
+    .form-group
+      = f.label :theft_alert_plan_id, class: "form-label"
```

Comment:
Do you think there is a reason admins need to be able to edit this?

Reply:
A scenario in which someone requests a plan change (potentially with accompanying changes to start/end time fields) is more plausible than changes being required to, say, `creator` or `stolen_record`.

Reply:
Fair. I can see the use of it for that. I also don't see a reason to enable editing creator or stolen_record